RELEASE/scripts/autoscend/quests/level_05.ash

```diff
@@ -62,6 +69,12 @@ boolean L5_haremOutfit()
 	{
 		return false;
 	}
+		
+	if (is_banished($phylum[goblin]) && get_property("screechCombats").to_int() > 0)
```

Comment:
there are only goblins in the harem, so if they are banished they will all come back, so adventuring is fine

unless you have a habitated monster, in which case you will only fight your habitated monster